firebase-functions/src/main/java/com/google/firebase/functions/HttpsCallableResult.kt

```diff
@@ -11,29 +11,20 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+package com.google.firebase.functions
 
-package com.google.firebase.functions;
-
-import androidx.annotation.Nullable;
-
-/** The result of calling a HttpsCallableReference function. */
-public class HttpsCallableResult {
-  // The actual result data, as generic types decoded from JSON.
-  private final Object data;
-
-  HttpsCallableResult(Object data) {
-    this.data = data;
-  }
-
+/** The result of calling a HttpsCallableReference function.  */
+class HttpsCallableResult internal constructor( // The actual result data, as generic types decoded from JSON.
+        private val data: Any?) {
   /**
    * Returns the data that was returned from the Callable HTTPS trigger.
    *
-   * <p>The data is in the form of native Java objects. For example, if your trigger returned an
+   *
+   * The data is in the form of native Java objects. For example, if your trigger returned an
```

Comment:
<p> was removed, is this fine?

Reply:
Yeah that's fine, but it's worth keeping an eye out for- if it removes anything more important.